src/Sentry/Protocol/ISpanContext.cs

```diff
@@ -26,12 +26,12 @@ public interface ISpanContext
         /// <summary>
         /// Operation.
         /// </summary>
-        string Operation { get; }
+        string Operation { get; set; }
```

Comment:
Should this be nullable? Or have some sort of default value?

Reply:
What's the use case for not having an operation? If we make it non-nullable are going to force users to set gibberish because they often have nothing to set?

Gut feeling is that we should always have an `operation` otherwise we're likely instrumenting something badly. Can we have it non-null (readonly also is not an option?) and we can change it later.

Reply:
if you don't have an op you're saying you don't know what you're instrumenting. That can't be right